requirements/app.in

```diff
@@ -59,8 +59,8 @@ marshmallow>=3
 marshmallow-polyfield
 marshmallow-sqlalchemy>=0.23.1
 webargs
-# Minimum version that correctly aligns time series that include NaN values
-uniplot>=0.7.0
+# Minimum version that supports datetime plotting features required for the new code
```

Comment:
Just out of curiosity: Would something break if someone used a version below `0.14.1`?

Reply:
Possibly more to the point:
```suggestion
# Minimum version that supports datetimes on the (x-)axis.
```

Reply:
You are correct to point this out, i just tested the new plotitng code with uniplot versions below `0.14.1` on python 3.8 through  3.11 and got: 
  - For uniplot versions from `0.7.0` to `0.11.0`, the x-axis labels display unclear numerical values:

 ```bash 
Beliefs for Sensor 'day-ahead prices' (ID 1).
  Data spans a day and starts at 2024-09-10 00:00:00+01:00.
┌────────────────────────────────────────────────────────────┐
│       ▞▀▄                                                  │ 
│      ▐   ▚                                                 │ 
│     ▗▘   ▐                                                 │ 
│     ▞     ▌                                                │ 15EUR/kWh
│    ▞      ▚                                     ▄          │ 
│   ▗▘      ▐                                   ▄▀▝▚         │ 
│   ▌        ▌                                ▄▀    ▀▖       │ 
│ ▄▀         ▚                               ▛       ▝▖      │ 
│▀           ▐                              ▐         ▚      │ 
│             ▚▖          ▗▖               ▗▘          ▚    ▗│ 10EUR/kWh
│              ▝▄▄▞▀▄▄   ▄▘▝▖              ▞            ▌   ▞│ 
│                     ▀▀▛   ▝▖            ▗▘            ▝▖ ▞ │ 
│                            ▝▖           ▞              ▐▗▘ │ 
│                             ▚          ▞                ▘  │ 
│                              ▚        ▞                    │ 
│                              ▝▄      ▞▘                    │ 5EUR/kWh
│                                ▀▄▄▄▄▀                      │ 
└────────────────────────────────────────────────────────────┘
        1,725,950,000,000,000,000           1,726,000,000,000,000,000
```
  - For uniplot versions from `0.12.1` to `0.14.0`, the x-axis labels show datetime values in a readable and informative format, using the full ISO 8601 representation (2024-09-10T05:48):
   
```bash
Beliefs for Sensor 'day-ahead prices' (ID 1).
  Data spans a day and starts at 2024-09-10 00:00:00+01:00.
┌────────────────────────────────────────────────────────────┐
│       ▞▀▄                                                  │ 
│      ▐   ▚                                                 │ 
│     ▗▘   ▐                                                 │ 
│     ▞     ▌                                                │ 15EUR/kWh
│    ▞      ▚                                     ▄          │ 
│   ▗▘      ▐                                   ▄▀▝▚         │ 
│   ▌        ▌                                ▄▀    ▀▖       │ 
│ ▄▀         ▚                               ▛       ▝▖      │ 
│▀           ▐                              ▐         ▚      │ 
│             ▚▖          ▗▖               ▗▘          ▚    ▗│ 10EUR/kWh
│              ▝▄▄▞▀▄▄   ▄▘▝▖              ▞            ▌   ▞│ 
│                     ▀▀▛   ▝▖            ▗▘            ▝▖ ▞ │ 
│                            ▝▖           ▞              ▐▗▘ │ 
│                             ▚          ▞                ▘  │ 
│                              ▚        ▞                    │ 
│                              ▝▄      ▞▘                    │ 5EUR/kWh
│                                ▀▄▄▄▄▀                      │ 
└────────────────────────────────────────────────────────────┘
  2024-09-10T02:49     2024-09-10T10:30   2024-09-10T18:10
          ██ day-ahead prices (NL transmission zone)
```
     
  - For version `0.14.1`, fewer datetime points are shown on the x-axis, with the ISO format truncated (no time component like T05:48):

```bash
Beliefs for Sensor 'day-ahead prices' (ID 1).
  Data spans a day and starts at 2024-09-10 00:00:00+01:00.
┌────────────────────────────────────────────────────────────┐
│       ▞▀▄                                                  │ 
│      ▐   ▚                                                 │ 
│     ▗▘   ▐                                                 │ 
│     ▞     ▌                                                │ 15EUR/kWh
│    ▞      ▚                                     ▄          │ 
│   ▗▘      ▐                                   ▄▀▝▚         │ 
│   ▌        ▌                                ▄▀    ▀▖       │ 
│ ▄▀         ▚                               ▛       ▝▖      │ 
│▀           ▐                              ▐         ▚      │ 
│             ▚▖          ▗▖               ▗▘          ▚    ▗│ 10EUR/kWh
│              ▝▄▄▞▀▄▄   ▄▘▝▖              ▞            ▌   ▞│ 
│                     ▀▀▛   ▝▖            ▗▘            ▝▖ ▞ │ 
│                            ▝▖           ▞              ▐▗▘ │ 
│                             ▚          ▞                ▘  │ 
│                              ▚        ▞                    │ 
│                              ▝▄      ▞▘                    │ 5EUR/kWh
│                                ▀▄▄▄▄▀                      │ 
└────────────────────────────────────────────────────────────┘
            04:00          10:00           16:00          22:00
          ██ day-ahead prices (NL transmission zone)
```

Reply:
So i guess I can update the uniplot version in the requirements to whichever version produces the preferred output v0.12.5 or v0.14.1.

Reply:
Then I'd keep the `==0.14.1` for the .txt requirements, but use `>=0.12.1` for the .in requirement.

Reply:
okay.